cypress/integration/account_settings/general/nickname.js

```diff
@@ -82,8 +82,8 @@ describe('Account Settings > Sidebar > General', () => {
         cy.get('#accountSettingsHeader > .close').click();
         cy.get('#sidebarHeaderDropdownButton').click();
 
-        cy.get('#manageMembers').should('be.visible');
-        cy.get('#manageMembers').click();
+        cy.get('#viewMembers').should('be.visible');
+        cy.get('#viewMembers').click();
```

Comment:
`user-1` by default don't have a permission to manage members on sample team/channel and so just use `viewMembers`